### ISSUE TITLE
update patient form (general information) UX

### ISSUE THREAD

**Issue by vault-developer** [HospitalRun/hospitalrun#393]
## 🚀 Feature Proposal
Update general information form UX
1. Put "Unknown" checkbox under "Date of Birth" field.
2. Do not hide "Date of Birth" field if "Unknown" is checked, only disable it.

## Motivation
Now we have input "Date of birth" and checkbox "Unknown"
1. It's not clear, that checkbox "unknown" is related to field "Date of birth".  User can suggest, that this checkbox is for "Patient type", because it's under this field.
2. If "Unknown" is checked, "Date of Birth" field is hidden. If someone will view such patient, there is no way to find out, what field does "Unknown" belong to.

![image](https://user-images.githubusercontent.com/28157258/86619003-4089d180-bfd3-11ea-98dc-091dc46464df.png)

P.S. I can work on this issue if you don't mind

**Comment by vault-developer** [comment]
sorry, It's for frontend repo

**Comment by tehKapa** [comment]
The date of birth isn't a mandatory field, so if you don't enter it, it is in fact already "Unknown".
@lauggh @jackcmeyer @fox1t opinions? does this new checkbox come in useful?

**Comment by lauggh** [comment]
Yes. I agree with the proposal here. While not required, it is useful as a checklist of questions during data entry or registration

**Comment by fox1t** [comment]
I agree too with the proposal. Actually this makes DOB mandatory. So let’s add also it to this proposal.

**Comment by blestab** [comment]
> Do not hide "Date of Birth" field if "Unknown" is checked, only disable it.

I am not sure about this proposal as checking "Unknown" shows the Approximate Age (at least in staging), which takes an integer and i thought the idea here was to calculate an approximate Date of Birth based on the Approximate Age?

![image](https://user-images.githubusercontent.com/5468546/86628502-43cc8f80-bfca-11ea-856f-4f5be32662da.png)

On Editing the patient, the Date of birth field now has a value

![image](https://user-images.githubusercontent.com/5468546/86628563-5cd54080-bfca-11ea-8e46-4592eee3a9a6.png)

Perhaps what we need is helper text to tell the user of the relationship between Date Of Birth, Unknown and Approximate Age fields?
